codegen/test/api/things.test.io/v1/types.go

```diff
@@ -21,6 +22,15 @@ type Paint struct {
 	Status PaintStatus `json:"status,omitempty"`
 }
 
+// GVK returns the GroupVersionKind associated with the resource type.
+func (Paint) GVK() schema.GroupVersionKind {
+	return schema.GroupVersionKind{
+		Group:   "things.test.io/v1",
```

Comment:
these groups aren't generating correctly, should be `things.test.io`

Reply:
Should this file be in VCS in general? It looks like its only created via tests, not codegen.